supabase/migrations/20240430155721_rename_profiles_plus_to_users_plus_and_include_profile_id.sql

```diff
@@ -0,0 +1,42 @@
+drop view if exists "public"."profiles_plus";
+
+create or replace view "public"."users_plus" as  SELECT users.id AS user_id,
+    users.email,
+    users.created_at,
+    users.updated_at,
+    profiles.primary_key AS profile_id,
```

Comment:
The view now includes profile ID. This allows us to update a user's by their profile ID instead of by their user ID. This means we can populate the profile ID in the audit log without having to select it from the DB after updating. This is needed because if the update fails, we do not retrieve the profile ID, meaning we can't populate it in the audit log, if it fails.